lib/tasks/sdr.rake

```diff
@@ -17,11 +19,11 @@ end
 namespace :sdr do
   desc 'Run Solr and GeoBlacklight for interactive development'
   task server: %i[environment] do |_t, _args|
-    require 'solr_wrapper'
-
     shared_solr_opts = { managed: true, verbose: true, persist: false, download_dir: 'tmp' }
     shared_solr_opts[:version] = Settings.SOLR_VERSION if Settings.SOLR_VERSION
 
+    FileUtils.rm_rf Settings.SOLR_INSTANCE_DIR
```

Comment:
Does this mean that locally every time we stand up the server we need to re-download Solr and re-import data?

Would it be worth putting this in a dedicated task you can use when you run into this issue?

Reply:
Solr doesn't get redownloaded, those files are safely in `tmp` rather than the `tmp/sdr-core-development` sub folder. When I rerun it seems to take ~2 extra seconds at most?

Reply:
Alright, sounds good to me then!